src/Microsoft.DotNet.Build.Tasks/PackageFiles/resources.targets

```diff
@@ -75,7 +75,21 @@
     <GenerateResourcesCode
         ResxFilePath="$(StringResourcesPath)" 
         OutputSourceFilePath="$(IntermediateResOutputFileFullPath)" 
-        AssemblyName="$(AssemblyName)" />
+        AssemblyName="$(AssemblyName)" >
+
+        <Output TaskParameter="NormalizedAssemblyName" PropertyName="_NormalizedAssemblyName" />
```

Comment:
While the normalization might work wouldn't it be easier to just allow projects to set the name via some other property so we can control it for a given project as necessary?